src/App.tsx

```diff
@@ -12,7 +14,7 @@ function App() {
       <BackendConnectionChecker />
 
       <Routes>
-        <Route path="/" element={<></>} />
+        <Route path="/createsighting" element={<CreateSighting />} />
```

Comment:
As discussed, we should draw attention to the front & backend paths at stand-up.
I think the best path here would be `"/sighting/create"`